ros/api/common/utils.py

```diff
@@ -44,23 +49,22 @@ def check_for_user():
         if username is None:
             abort(403, message="Username doesn't exist")
 
-        return username
+        return ident, username
 
-    def check_for_system(inventory_id):
+    def check_for_system(ident, inventory_id):
         if not is_valid_uuid(inventory_id):
             abort(404, message='Invalid inventory_id, Id should be in form of UUID4')
 
-        system = System.query.filter(
-            System.inventory_id == inventory_id
-        ).first()
+        system_id = system_ids_by_org_id(ident['org_id']).filter(
```

Comment:
nit, can we name the var to system_ids ?

Reply:
As I applied `first()` which returns single element, I think it is better to keep `system_id` as var. Thoughts?